### PR TITLE
updated hammer image

### LINKED ISSUE #142: Hammercli image build fails in centos7
Reported by https://projects.engineering.redhat.com/browse/SCALELAB-1162

### PR DESCRIPTION
**Pull request by mukrishn**
# Description

I could reproduce the issue in centos7, updated them to use centos:8. 

Fixes #142 

## Type of change

Please select the appropriate options:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] This change is a documentation update

## Testing

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Tested locally
- [ ] Test B

**Test Configuration**:

- Versions: 
- Hardware: Dell R640

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] If a change is adding a feature, it should require a change to the README.md and the review should catch this.
- [ ] If the change is a fix, it should have an issue. The review should make sure the comments state the issue (not just the number) and it should use the keywords that will close the issue on merge.
- [ ] A change should not be merged unless it passes CI or there is a comment/update saying what testing was passed.
- [ ] PRs should not be merged unless positively reviewed.

### CODE REVIEW

**Comment by mukrishn** [comment]
@akiselev1 could you test this patch and let us know ?

File: ansible-ipi-install/roles/bootstrap/files/Dockerfile [hunk 1]
```diff
@@ -1,16 +1,16 @@
-FROM docker.io/centos:centos7
+FROM centos:8
```
Comment (smalleni) [inline ansible-ipi-install/roles/bootstrap/files/Dockerfile:1]: Is this using quay.io then?

File: ansible-ipi-install/roles/bootstrap/files/Dockerfile [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 ARG foreman_url=https://foreman.example.com
 RUN yum install -y epel-release \
-    https://yum.theforeman.org/releases/2.1/el7/x86_64/foreman-release.rpm \
-    centos-release-scl-rh \
-    rh-ruby25-ruby
+        python3
 
-RUN yum install -y tfm-rubygem-hammer_cli \
-    tfm-rubygem-hammer_cli_foreman
+RUN yum install -y https://yum.theforeman.org/releases/2.1/el8/x86_64/foreman-release.rpm
```
Comment (smalleni) [inline ansible-ipi-install/roles/bootstrap/files/Dockerfile:6]: Lets condense into one RUN/yum command, would result in fewer layers in the image.

**Comment by akiselev1** [comment]
> @akiselev1 could you test this patch and let us know ?

Yes, the fix works for me. I can try the final version as well - let me know.

File: ansible-ipi-install/roles/bootstrap/files/Dockerfile (hunk 1 above)
Comment (mukrishn) [reply ansible-ipi-install/roles/bootstrap/files/Dockerfile:1]: I think it still goes to docker.io

File: ansible-ipi-install/roles/bootstrap/files/Dockerfile (hunk 2 above)
Comment (smalleni) [reply ansible-ipi-install/roles/bootstrap/files/Dockerfile:6]: Move to dnf as it is centos8

**Review by smalleni — CHANGES_REQUESTED** [review]

**Comment by akiselev1** [comment]
The new version works fine as well.

**Review by smalleni — APPROVED** [review]

**Pull request merged**